rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtils.java

```diff
@@ -117,6 +119,9 @@ protected void prePopulateRDTPatientFields(Patient patient, JSONObject field) th
         } else if (PATIENT_AGE.equals(key)) {
             field.put(JsonFormUtils.VALUE, patient.getAge());
         }
+        else if (CovidConstants.FormFields.FACILITY_NAME.equals(key) || CovidConstants.FormFields.HEALTH_FACILITY_NAME.equals(key) || CovidConstants.FormFields.OTHER_HEALTH_FACILITY_TYPE_NAME.equals(key)) {
```

Comment:
https://github.com/onaio/rdt-standard/issues/349#issuecomment-701148554 this only populates the RDT test form. `Symptom and history form` and `Sample collection form` will also need population.

Reply:
it works in all three forms